games/sudoku-solver/Main.java

```diff
@@ -0,0 +1,16 @@
+class Main {
+  public void playSudoku(Sudoku sudoku) {
+    int[][] board = new int[9][9];
```

Comment:
Shouldn't this and next 4 lines be part of Sudoku constructor. Why are we doing these here ?

Reply:
Agree, fixed here e18c92a